packages/codemod/src/lib/transform.ts

```diff
@@ -26,7 +26,11 @@ function buildCommand(
   let command = `${jscodeshift} -t ${codemodPath} ${targetPath} \
     --parser tsx \
     --ignore-pattern="**/node_modules/**" \
-    --ignore-pattern="**/.*/**"`;
+    --ignore-pattern="**/.*/**" \
+    --ignore-pattern="**/dist/**" \
```

Comment:
Slight risk re: `dist` and `build`, folks could... actually have a package in a dir named `build` that uses AI SDK for something, but that seems unlikely vs the value we gain from removing overhead of unnecessarily processing output files in such a dir.